modules/ROOT/pages/intro-mule-message.adoc

```diff
@@ -1,9 +1,14 @@
-= Introduction to Mule 4: The Mule Message
+= Introduction to Mule 4: Mule Message
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Mule 4 includes a simplified Mule message model in which each Mule event has a message and variables associated with it. A Mule message is composed of a payload and its attributes (metadata, such as file size). Variables (`vars`) hold arbitrary user information such as operation results, auxiliary values, and so on. This simplified message model makes it easier to work with data in a consistent way across connectors without overwriting information.
+Mule 4 includes a simplified Mule message model in which each Mule event has a message and variables associated with it. A Mule message is composed of: +
+
+* A payload and its attributes (metadata, such as file size).
+* Variables (`vars`) hold arbitrary user information such as operation results, auxiliary values, and so on. In general, operations do not automatically propagate variables to other components, like a listener. For example, if you want variables of a flow 1 available in a flow 2 you would need to explicitly send them as part of the message.
```

Comment:
Beware: flor-ref does propagate variables (maybe adding a clarification over this to avoid misunderstandings)

Reply:
Updated.